src/components/Lightbox.tsx

```diff
@@ -62,24 +77,33 @@ type LightboxStyle = {
 type Props = {
   index?: number
   onClose: () => void
+  onDelete?: (imageIndex: number) => void
   photos: string[]
   title?: string
   description?: string
   content?: JSX.Element
   style?: LightboxStyle
 }
 
-const Lightbox = ({ index, onClose, photos, title, description, content, style }: Props) => (
-  <ImageView
-    images={photos.map((photo) => ({ uri: photo }))}
-    imageIndex={index ?? 0}
-    visible={index !== undefined}
-    swipeToCloseEnabled={false}
-    onRequestClose={onClose}
-    HeaderComponent={getLightboxHeaderComponent(photos.length, onClose)}
-    FooterComponent={getLightboxFooterComponent(title, description, content, style)}
-  />
-)
+const Lightbox = ({ index, onClose, onDelete, photos, title, description, content, style }: Props) => {
+  const initialIndex = index ?? 0
```

Comment:
Of dit om consistent te blijven?
```suggestion
  const initialImageIndex = index ?? 0
```